modules/extensions-mule-support/src/test/java/org/mule/runtime/module/extension/mule/internal/loader/parser/Utils.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.extension.mule.internal.loader.parser;
+
+import static org.mule.runtime.api.functional.Either.right;
+import static org.mule.runtime.api.meta.model.parameter.ParameterGroupModel.DEFAULT_GROUP_NAME;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import org.mule.runtime.ast.api.ComponentAst;
+import org.mule.runtime.ast.api.ComponentParameterAst;
+
+public class Utils {
```

Comment:
I think it could be useful for anyone else trying to write more tests that could benefit from these utils. It doesn't need to be a huge description. Maybe the Javadoc for the utils class itself would not be very useful as it will be self explanatory. But for the methods I think it could make sense.

Reply:
done